README.md

```diff
@@ -10,15 +10,14 @@
     - After making changes in the blog run `docker start jekyll -i` to rebuild and watch the blog
     
 ## Setting up Jekyll on your Mac or Linux
-Execute following commands:
-    
-    Make sure you have the ruby headers installed.
-    for Ubuntu and other distributions using Debian style packaging
-    - sudo apt-get install ruby2.3-dev(make sure this matches your ruby version)
+You need to have atleast Ruby 2.4.0 installed, this can be easily accomplished by using [rvm](https://rvm.io).
+
+Execute the following commands:
     
-    then
-    - `(sudo) gem install bundler`
-    - `bundler install`
+    $ rvm install 2.4.0
+    $ rvm use 2.4.0
+    $ sudo gem install bundler
```

Comment:
`sudo` is not required for everyone, and best practice is not to use it, so I would put it between parenthesis like before.